sim/core/apl_helpers.go

```diff
@@ -127,6 +128,18 @@ func (rot *APLRotation) GetAPLICDAura(sourceUnit UnitReference, auraId *proto.Ac
 	return aura
 }
 
+func (rot *APLRotation) GetAPLTrinketProcAuras(statTypesToMatch []stats.Stat, warnIfNoneFound bool) []*StatBuffAura {
+	unit := rot.unit
+	character := unit.Env.Raid.GetPlayerFromUnit(unit).GetCharacter()
+	matchingAuras := character.GetMatchingTrinketProcAuras(statTypesToMatch)
+
+	if (len(matchingAuras) == 0) && warnIfNoneFound {
+		rot.ValidationWarning("No trinket proc buffs found for: %s", StringFromStatTypes(statTypesToMatch))
+	}
```

Comment:
Is it feasible to "warn" the list of auras that are found, like the cast ones are doing?

Reply:
This should be implemented now.